src/NServiceBus.AcceptanceTests/EndpointTemplates/Requires.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    static class Requires
```

Comment:
Does this need to have methods for `NativePubSubSupport`, `CrossQueueTransactionSupport` and `NativeDeferralSupport`?